core/src/main/java/org/apache/iceberg/CatalogProperties.java

```diff
@@ -29,6 +29,7 @@ private CatalogProperties() {}
   public static final String WAREHOUSE_LOCATION = "warehouse";
   public static final String TABLE_DEFAULT_PREFIX = "table-default.";
   public static final String TABLE_OVERRIDE_PREFIX = "table-override.";
+  public static final String VIEW_DEFAULT_PREFIX = "view-default.";
```

Comment:
Thanks @ebyhr for working on this. I also raised a similar [PR](https://github.com/apache/iceberg/pull/11025) which I think is not required anymore. Do you think we should also add `"view-override."` ?

Reply:
That probably makes sense since we have the table-default as well, but i have no problem this being in a followup